### ISSUE TITLE
Унатхи и гсб

### ISSUE THREAD

**Issue by RiEviL4536** [TauCetiStation/TauCetiClassic#4448]
#### Подробное описание проблемы
У унатхов закрыли (по крайней мере у меня) ГСБ, но периодически закидывает за него. На форумах не написано о изменениях в профах.

#### Что должно было произойти
открыться професия
#### Что произошло на самом деле
закрыта из-за рассы

**Comment by Sakuya-Izayoi** [comment]
Нет, унатхам ГСБ закрыт. Баг в том, что иногда за него унатхов закидывает.

**Comment by LudwigVonChesterfield** [comment]
преференсы людей которые были до изменения ограничений не сбрасываются и не меняются, в итоге люди с префами на ГСБ всё ещё могут зайти за ГСБ

**Comment by Udokun** [comment]
@TEXH4Pb @RichardJones1 подозреваю, что давно неактуально.

**Comment by RichardJones1** [comment]
@volas ты не фиксил это в #14238 ?
